stable/redis/ci/default-values.yaml

```diff
@@ -1,354 +1 @@
-
-  ## Redis Slave pod/node affinity/anti-affinity
-  ##
-  affinity: {}
-
-  ## Configure extra options for Redis Slave liveness and readiness probes
-  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes)
-  ##
-  # livenessProbe:
-  #   enabled: true
-  #   initialDelaySeconds: 30
-  #   periodSeconds: 10
-  #   timeoutSeconds: 5
-  #   successThreshold: 1
-  #   failureThreshold: 5
-  # readinessProbe:
-  #   enabled: true
-  #   initialDelaySeconds: 5
-  #   periodSeconds: 10
-  #   timeoutSeconds: 10
-  #   successThreshold: 1
-  #   failureThreshold: 5
-
-  ## Redis slave Resource
-  # resources:
-  #   requests:
-  #     memory: 256Mi
-  #     cpu: 100m
-
-  ## Redis slave selectors and tolerations for pod assignment
-  # nodeSelector: {"beta.kubernetes.io/arch": "amd64"}
-  # tolerations: []
-
-  ## Use an alternate scheduler, e.g. "stork".
-  ## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
-  ##
-  # schedulerName:
-
-  ## Redis slave pod Annotation and Labels
-  # podLabels: {}
-  # podAnnotations: {}
-
-  ## Redis slave pod Security Context
-  # securityContext:
-  #   enabled: true
-  #   fsGroup: 1001
-  #   runAsUser: 1001
-
-## Prometheus Exporter / Metrics
-##
-metrics:
-  enabled: false
-
-  image:
-    registry: docker.io
-    repository: oliver006/redis_exporter
-    tag: v0.20.2
-    pullPolicy: IfNotPresent
-    ## Optionally specify an array of imagePullSecrets.
-    ## Secrets must be manually created in the namespace.
-    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
-    ##
-    # pullSecrets:
-    #   - myRegistrKeySecretName
-
-  service:
-    type: ClusterIP
-    ## Use serviceLoadBalancerIP to request a specific static IP,
-    ## otherwise leave blank
-    # loadBalancerIP:
-    annotations:
-      prometheus.io/scrape: "true"
-      prometheus.io/port: "9121"
-
-  ## Metrics exporter resource requests and limits
-  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
-  ##
-  # resources: {}
-
-  ## Extra arguments for Metrics exporter, for example:
-  ## extraArgs:
-  ##   check-keys: myKey,myOtherKey
-  # extraArgs: {}
-
-  ## Metrics exporter labels and tolerations for pod assignment
-  # nodeSelector: {"beta.kubernetes.io/arch": "amd64"}
-  # tolerations: []
-
-  ## Metrics exporter pod Annotation and Labels
-  # podAnnotations: {}
-  # podLabels: {}
-
-##
-## Init containers parameters:
-## volumePermissions: Change the owner of the persist volume mountpoint to RunAsUser:fsGroup
-##
-volumePermissions:
-  image:
-    registry: docker.io
-    repository: bitnami/minideb
-    tag: latest
-    pullPolicy: IfNotPresent
-
-## Redis config file
-## ref: https://redis.io/topics/config
-##
-configmap: |-
-  # maxmemory-policy volatile-lru
+# Leave this file empty to ensure that CI runs builds against the default configuration in values.yaml.
```

Comment:
I think we can just delete this file? Otherwise the CI will try to run it twice because I believe it always tests with the defaults, and additionally tests with any files found in this directory.

Reply:
The chart-testing library only runs with the default `values.yaml` if no other values files are present in the `ci` directory (sometimes this is necessary if the user is expected to fill in some value).

https://github.com/helm/chart-testing/blob/46dc82e9723043e327bb01586e5f133a10b2e801/pkg/chart/chart.go#L329-L332

Reply:
You're right, thanks! I think it's this way because sometimes default values don't work out of the box (e.g. when a token is needed).